### PR TITLE
Status bar tests

### PR DESCRIPTION
**Pull request by Draggu**
**Stack**:
- #32
- #31
- #30 ⬅

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: test-support/scarb.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import commandExists from "command-exists";
+
+let isScarbAvailableState: null | boolean = null;
+
+export const isScarbAvailable = async () =>
+  (isScarbAvailableState ??= await commandExists("scarb")
+    .then(() => true)
+    .catch(() => false));
```
Comment (mkaput) [inline test-support/scarb.ts:8]:
```suggestion
import { sync as commandExistsSync } from "command-exists";

export const isScarbAvailable = commandExistsSync("scarb");
```

**Review by mkaput — CHANGES_REQUESTED** [review]

File: test-support/scarb.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { sync as commandExists } from "command-exists";
+
+export const isScarbAvailable = () => commandExists("scarb");
```
Comment (mkaput) [inline test-support/scarb.ts:3]:
```suggestion
export const isScarbAvailable = commandExists("scarb");
```

discussed offline 😆

**Review by mkaput — CHANGES_REQUESTED** [review]

**Review by mkaput — APPROVED** [review]

**Review by piotmag769 — APPROVED** [review]

File: test-support/scarb.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { sync as commandExists } from "command-exists";
+
+export const isScarbAvailable = commandExists("scarb");
```
Comment (Arcticae) [inline test-support/scarb.ts:3]:
```suggestion
export const scarbIsAvailable = commandExists("scarb");
```

Comment (mkaput) [reply test-support/scarb.ts:3]: tbh looks bad to my eyes

**Pull request merged**